Automattic/newspack-listings
# refactor: use material icons with Newspack color

### All Submissions:






### Changes proposed in this Pull Request:



A small refactor which replaces hard-coded Material Design SVGs with icons from the [Material Design Icons NPM package](https://www.npmjs.com/package/material-design-icons). Also ensures that all icons are displayed with the Newspack brand color where appropriate.

Also updates some docblock comments to maintain consistency between the use of "WordPress" vs. "External" dependencies.

### How to test the changes in this Pull Request:

1. Check out this branch, run `npm ci && npm run build`.
2. On a post or page, add a Curated List block. Confirm that the block icon is now blue to match other Newspack icons:

<img width="332" alt="Screen Shot 2021-03-19 at 4 52 06 PM" src="https://user-images.githubusercontent.com/2230142/111849557-79ed3d80-88d3-11eb-9894-25996f90d559.png">

3. Select "Specific Listings" mode. Add blocks for each listing type to the list. Confirm that the individual icons for each different type are now Newspack Blue in both the block inserters and the placeholders:

<img width="340" alt="Screen Shot 2021-03-19 at 4 53 02 PM" src="https://user-images.githubusercontent.com/2230142/111849639-b3be4400-88d3-11eb-8af0-e68d15e3dbbe.png">
<img width="818" alt="Screen Shot 2021-03-19 at 4 52 55 PM" src="https://user-images.githubusercontent.com/2230142/111849646-b7ea6180-88d3-11eb-8ff6-f17b6a3f9ef4.png">


### Other information:
